libs/utils/utils.go

```diff
@@ -0,0 +1,12 @@
+package utils
```

Comment:
Can this be more focused, like "sortutil"? (we have textutil as well, for ex)

Reply:
It's hard to predict whether future additions would be related to sorting? We generalizing from one example here. But it's good to have a place for small commonly used utilities without being forced to create a dedicated package for each.

Shorter name is easier to type. I'd actually even go with "util".